inputs_supply.md

```diff
@@ -0,0 +1,167 @@
+| Logit exponents | GCAM requires the user to specify the logit exponents that determine the substitutability between technologies. | Assumption |  | Specified by sector and subsector | N/A |
+| Share weight interpolation rules | These rules dictate how share weights (GCAM's calibration parameter) are specified in future years. | Assumption |  | Specified by sector and subsector | N/A |
+| Cost of conversion technologies | Cost of production for conversion technologies | External data | Various | Specified by technology and year |  1975$/GJ |
+| Capital cost | Overnight capital cost of electricity generation technologies | External data | Various | Specified by technology and year |  1975$/kW |
+| Fixed O&M costs | Fixed operating and maintenance (O&M) costs for electricity generation technologies | External data | Based on Annual Energy Outlook | Specified by technology and year |  1975$/kW/yr |
+| Variable O&M costs | Variable operating and maintenance (O&M) costs for electricity generation technologies | External data | Based on Annual Energy Outlook | Specified by technology and year |  1975$/MWh |
+| Capacity factor | Ratio of generation to capacity | Assumption |  | Specified by technology and year |  Unitless |
+| Fixed charge rate | Factor used to levelize capital cost | Assumption |  | Specified by technology |  Unitless |
+| Default efficiencies | Default amount of output produced per unit of input; can be overwritten by region-specific information derived from historical data | Assumption | | Specified by technology and year | GJ per GJ |
+| Default input-output coefficients | Default amount of input required per unit of output produced; can be overwritten by region-specific information derived from historical data | Assumption | | Specified by technology and year |  GJ per GJ |
+| Resource supply curves | Mapping between cost and resource extraction. Resource extraction is cumulative for deplatable resources and annual for renewable resources | External data | Various | Specified by resource and year |  EJ for extraction, 1975$/GJ for cost |
+
+
+Table 1: External inputs used for supply of energy<sup>[1](#table_footnote1)</sup>
+
+Note that for the Shared Socioeconomic Pathways (SSPs), different inputs are used for some variables. See [SSPs](ssp.html) for more information.
+
+#### Data
+
+Throughout GCAM, the number in the name of assumption file indicates to which sector the file applies. Files with `A10` through `A20` in the name are assumptions for resources. Files with `A21` and `A22` in the name are assumptions for refining and gas processing. Files with `A23` in the name are assumptions for electricity, `A24` is for heat, and `A25` is for hydrogen. Files with `A26` in the name are assumptions for transmission and distribution.
```

Comment:
For completeness, add:  "Files with `A61` in the name contain assumptions about carbon storage.